src/lib/registration/url_helpers.rb

```diff
@@ -119,6 +120,13 @@ def self.reg_url_at_installation
       slp_service_url
     end
 
+    # get registration URL from AutoYaST configuration file
+    def self.reg_url_from_autoyast_config
+      server = ::Registration::Storage::Config.instance.reg_server
+      return server if server && !server.empty?
+      SUSE::Connect::Config::DEFAULT_CONFIG_FILE
```

Comment:
JFYI: YaST should only use the constants/methods from the [connect/yast.rb](https://github.com/SUSE/connect/blob/master/lib/suse/connect/yast.rb) file. The rest is considered unstable and can be changed by the SCC team anytime...

And the constant should be the actually the `DEFAULT_URL`, not the config file.

So the correct value is `SUSE::Connect::YaST::DEFAULT_URL`. Please adapt the tests as well.